models/pardot/model.sql

```diff
@@ -36,13 +36,13 @@ create or replace view pardot_analysis.visitoractivity_types_meta as (
   select 31, 'UserVoice Suggestion' union all
   select 32, 'UserVoice Comment' union all
   select 33, 'UserVoice Ticket' union all
-  select 34, 'Video Watched (≥ 75% watched)'
+  select 34, 'Video Watched (>= 75% watched)'
```

Comment:
Created an issue out of this: https://github.com/analyst-collective/models/issues/9